python/cuml/__init__.py

```diff
@@ -44,6 +44,8 @@
 
 from cuml.random_projection.random_projection import GaussianRandomProjection, SparseRandomProjection, johnson_lindenstrauss_min_dim
 
+from cuml.preprocessing.utils import train_test_split
```

Comment:
For ease of users finding tools (and potentially dask compability depending on the dask-ml internals), we'll probably want to mimic scikit-learn's preprocessing structure as much as possible.

Perhaps we should design for eventually having other `model_selection` tools, making the natural place functionality for this `cuml.preprocessing.model_selection`.

Reply:
Agreed. I'd thought about it, but wasn't sure if we were planning on supporting enough model selection tools to warrant that